mysql/tests/compose/mysql.yaml

```diff
@@ -7,6 +7,9 @@ services:
       - MYSQL_ALLOW_EMPTY_PASSWORD=1
     ports:
       - "${MYSQL_PORT}:3306"
+    volumes:
+      # This is a supported conf location that isn't used by the image.
+      - ${MYSQL_CONF_PATH}:/etc/my.cnf
```

Comment:
Note to reviewers: list of supported config files are documented [here](https://dev.mysql.com/doc/refman/8.0/en/option-files.html) (table 4.2).